.gitignore

```diff
@@ -1,3 +1,6 @@
 /vendor
-/src/Collect/Support/Carbon.php
+/tests/files/Support/Carbon.php
```

Comment:
Why are we git ignoring these?

Reply:
They are only present to support our ability to run tests and don't have an effect on the actual function of the package.  Since both our `upgrade.sh` and CI builds already build these assets, it doesn't make sense to me to have them versioned and add  unnecessary files to the repo.

Reply:
Are the tests git ignored too? If not, I don't know whether there's any reason to exclude some files which are test-only but not others.

Reply:
Ok, that's a reasonable argument.  👍   I was following the seemigly failed intention of ignoring the import of `Carbon`.